_docs/style/interactive-states.md

```diff
@@ -0,0 +1,348 @@
+
+You are not required to apply all unselected styles to an input or control. For example, in the default state, a text input receives only the unselected idle style. Do not apply any of the other three unselected styles to a text input.
+
+Only multi-select controls, like checkboxes, use the unselected focused state. Single-select controls such as radios, do not have an unselected focused state, because they are automatically selected when focused.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Focus state composition"] [class:"image bg-light border"] %}
+
+<img src="{{site.url}}/assets/images/style/interactive-states/interactive-states-unselected-styles.png" width="539"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Selected styles
+
+Selected styles are most often used for selectable controls, like checkboxes. However, some selected styles can also be used for other controls like text inputs or drop-down selectors.
+
+You are not required to apply all selected styles to an input or control. For example, text inputs use the selected focused style for the focused state, but they do not use any of the other three styles.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Focus state composition"] [class:"image bg-light border"] %}
+
+<img src="{{site.url}}/assets/images/style/interactive-states/interactive-states-selected-styles.png" width="538"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Selected with fill
+
+Apply selected fill styles to elements that benefit from a more apparent selected state, including checkboxes, radios, switches, and toggles.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Focus state composition"] [class:"image bg-light border"] %}
+
+<img src="{{site.url}}/assets/images/style/interactive-states/interactive-states-selected-with-fill.png" width="519"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Errored styles
+
+In general, apply hovered states to selectable controls. For example, checkboxes and radios have a hover state, but text inputs do not.
```

Comment:
Let's remove this from here.  Not sure why I put this note in the errored section of the sketch file.  Most likely just a brain dump.

Anyway, I included this information in the best practices section.

Reply:
Done.